core/src/Platforms/uap/UapCryptographyManager.cs

```diff
@@ -72,15 +72,24 @@ public string CreateSha256Hash(string input)
             {
                 return null;
             }
-            
+
             IBuffer inputBuffer = CryptographicBuffer.ConvertStringToBinary(input, BinaryStringEncoding.Utf8);
 
             var hasher = HashAlgorithmProvider.OpenAlgorithm("SHA256");
             IBuffer hashed = hasher.HashData(inputBuffer);
```

Comment:
consider calling the method below instead of duplicating

Reply:
fixed